stylist_agents/remote_a2a/outfit_agent/agent.py

```
@@ -0,0 +1,91 @@
+from server.services.rules_engine import (
+    get_eligible_items as _get_eligible_items,
+    score_outfit as _score_outfit,
+    fetch_rules_data,
+)
+
+
+# Cache rules data at module level (reference data, rarely changes)
+_rules_cache: dict | None = None
+
+
+def _get_rules():
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_get_rules`

(ANN202)